contributing/pull-request-lifecycle.md

```diff
@@ -0,0 +1,46 @@
+# Pull Request Submission and Lifecycle
+
+__Note for internal service teams__: The pull request lifecycle below will be
+roughly the same for internal teams, but you will not need to fork the repo to
+create pull requests, and our recommendation is to get a review from one other
+member of your service team, in addition to a review from one member of the
+provider team.
+
+We appreciate direct contributions to the provider codebase. Here's what to
+expect:
+
+* For pull requests that follow the guidelines, we will proceed to reviewing
+  and merging, following the provider team's review schedule. There may be some
+  internal or community discussion needed before we can complete this.
+* Pull requests that don't follow the guidelines will be commented with what
+  they're missing. The person who submits the pull request or another community
+  member will need to address those requests before they move forward.
+
+## Pull Request Lifecycle
+
+1. [Fork the GitHub repository](https://help.github.com/en/articles/fork-a-repo),
```

Comment:
Might be worth linking to our resource checklist here.

Reply:
👍 definitely agree. I forgot to mention this in the changes I had made, but I did add a link to our checklists in the next bullet.